app/pages/admin/statistics/activities.vue

```diff
@@ -0,0 +1,439 @@
+    accessorKey: 'p25',
+    header: 'P25',
+  },
+  {
+    accessorKey: 'median',
+    header: 'Médiane',
+  },
+  {
+    accessorKey: 'p75',
+    header: 'P75',
+  },
+  {
+    accessorKey: 'evolution',
+    header: 'Évolution',
+  }
+]
+
+// Get available activities from all activities (not filtered by presence metrics)
+const availableActivities = computed(() => {
+  if (!selectedProfile.value?.club.presencesEnabled) {
+    return [];
+  }
+
+  // Get excluded activities from club settings
+  const excludedActivityUuids = selectedProfile.value?.club.settings?.excludedActivitiesFromOpeningDays?.map((a: Activity) => a.uuid) || [];
+
+  return allActivities.value
+    .filter(activity => {
+      // Only show enabled activities that are not excluded from opening days
+      return activity.isEnabled && !excludedActivityUuids.includes(activity.uuid);
+    })
+    .map(activity => ({
+      label: activity.name,
+      value: activity.name
+    }));
+});
+
+// Filter presence metrics for the selected activity
+const activityPresenceMetrics = computed(() => {
+  if (!presenceMetrics.value || !selectedActivity.value) {
+    return undefined;
+  }
+
+  // Filter childMetrics to find the specific activity
+  const activityMetric = presenceMetrics.value.childMetrics?.find(
+    metric => metric.name === selectedActivity.value
+  );
+
+  return activityMetric;
+});
+
+const activityPresenceMetricsPreviousSeason = computed(() => {
+  if (!presenceMetricsPreviousSeason.value || !selectedActivity.value) {
+    return undefined;
+  }
+
+  // Filter childMetrics to find the specific activity
+  const activityMetric = presenceMetricsPreviousSeason.value.childMetrics?.find(
+    metric => metric.name === selectedActivity.value
+  );
+
+  return activityMetric;
+});
+
+// Check if selected activity has data for current date range
+const hasActivityData = computed(() => {
+  return activityPresenceMetrics.value &&
+    activityPresenceMetrics.value.childMetrics &&
+    activityPresenceMetrics.value.childMetrics.length > 0;
+});
+
+const activityStats = computed(() => {
+  let response = {
+    loading: true,
+    currentSeason: 0,
+    previousSeason: 0,
+    isIncreasing: false,
+    totalValue: 0
+  }
+
+  if (activityPresenceMetrics.value) {
+    response.loading = false;
+    // Sum up all the total values from each day
+    const currentSeasonTotal = activityPresenceMetrics.value.childMetrics?.reduce((sum, day) => {
+      return sum + parseInt(day.values?.total || '0');
+    }, 0) || 0;
+
+    response.currentSeason = currentSeasonTotal;
+    response.totalValue = currentSeasonTotal;
+  }
+
+  if (activityPresenceMetricsPreviousSeason.value) {
+    // Sum up all the total values from each day for previous season
+    const previousSeasonTotal = activityPresenceMetricsPreviousSeason.value.childMetrics?.reduce((sum, day) => {
+      return sum + parseInt(day.values?.total || '0');
+    }, 0) || 0;
+
+    response.previousSeason = previousSeasonTotal;
+  }
```

Comment:
Code is repeating, maybe find a simpler method, also the total can be obtain directly (not in the childMetrics